tests/Unit/NumericalAlgorithms/LinearSolver/Test_InnerProduct.cpp

```diff
@@ -0,0 +1,36 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DenseVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "NumericalAlgorithms/LinearSolver/InnerProduct.hpp"
+#include "Utilities/TMPL.hpp"
+
+class DataVector;
```

Comment:
Don't you need an include for this instead?

Reply:
I think not, only instantiating `DenseVector`s explicitly. `DataVector` is only used in the `Variables`.